src/app/sitemap.ts

```diff
@@ -1,9 +1,14 @@
-import { getSiteUrl, JUMPER_LEARN_PATH, pages } from '@/const/urls';
+import {
+  getSiteUrl,
+  JUMPER_LEARN_PATH,
+  JUMPER_SWAP_PATH,
+  pages,
+} from '@/const/urls';
 import type { ChangeFrequency, SitemapPage } from '@/types/sitemap';
 import type { BlogArticleData, StrapiResponse } from '@/types/strapi';
 import type { MetadataRoute } from 'next';
+import { getChainsQuery } from 'src/hooks/useChains';
 import { getArticles } from './lib/getArticles';
-import { locales } from 'src/i18n';
 
 function withoutTrailingSlash(url: string) {
```

Comment:
Can you reuse the same function as for the blog and make it a util pls @dennyscode ?